### ISSUE TITLE
unexpected keyword argument 'private_key'

### ISSUE THREAD

**Issue by BroOtti** [Keeper-Security/Commander#767]
I just updated keeper commander from v16.6.6, and now I'm getting the error executing command ssh with a sshKeys type entry:

```
add_ssh_key() got an unexpected keyword argument 'private_key'
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/keepercommander/cli.py", line 522, in loop
    result = do_command(params, command)
  File "/usr/local/lib/python3.10/dist-packages/keepercommander/cli.py", line 366, in do_command
    result = command.execute_args(params, args, command=orig_cmd)
  File "/usr/local/lib/python3.10/dist-packages/keepercommander/commands/base.py", line 351, in execute_args
    return self.execute(params, **d)
  File "/usr/local/lib/python3.10/dist-packages/keepercommander/commands/connect.py", line 312, in execute
    to_remove = add_ssh_key(private_key=private_key, passphrase=passphrase, key_name=record.title)
TypeError: add_ssh_key() got an unexpected keyword argument 'private_key'
An unexpected error occurred: add_ssh_key() got an unexpected keyword argument 'private_key'. Toggle debug to print traceback
```

OS: Ubuntu 22.04
Python 3.10.4
Keeper Commander v16.6.8

Output of `pip3 list`:
```
Package                     Version
--------------------------- -------------
argon2-cffi                 21.3.0
argon2-cffi-bindings        21.2.0
asciitree                   0.3.3
bcrypt                      3.2.0
certifi                     2021.10.8
cffi                        1.15.0
charset-normalizer          2.0.12
colorama                    0.4.4
command-not-found           0.3
construct                   2.10.54
cryptography                36.0.2
dbus-python                 1.2.18
distro-info                 1.1build1
ecdsa                       0.17.0
future                      0.18.2
idna                        3.3
importlib-metadata          4.11.3
keeper-secrets-manager-core 16.2.2
keepercommander             16.6.8
lxml                        4.8.0
netifaces                   0.11.0
pip                         22.1.2
prompt-toolkit              3.0.29
protobuf                    3.20.1
pycparser                   2.21
pycryptodomex               3.14.1
PyGObject                   3.42.0
pykeepass                   4.0.1
pyperclip                   1.8.2
PySocks                     1.7.1
python-apt                  2.3.0+ubuntu2
python-dateutil             2.8.2
PyYAML                      5.4.1
requests                    2.27.1
setuptools                  59.6.0
six                         1.16.0
tabulate                    0.8.9
ubuntu-advantage-tools      27.8
ufw                         0.36.1
urllib3                     1.26.9
wcwidth                     0.2.5
wheel                       0.37.1
zipp                        3.8.0
```

**Comment by sk-keeper** [comment]
Thank you for reporting this issue. 
It is fixed in the master branch. Windows and MacOS binaries are fixed as well.
The official PyPI release that fixes this issue will be 16.6.9

**Comment by BroOtti** [comment]
Can confirm the issue is fixed with release 16.6.9
